src/formatDate.js

```diff
@@ -50,7 +50,43 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const splitedDate = date.split(fromFormat[3]);
+  const dateObject = {};
+  const dateResult = [];
+
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    if (fromFormat[i] === 'DD') {
+      dateObject.DD = splitedDate[i];
+    } else if (fromFormat[i] === 'MM') {
+      dateObject.MM = splitedDate[i];
```

Comment:
I think you don't need to check if fromFormat[i] is equal to 'DD' or 'MM', you can just add it to dateObject